mesonbuild/modules/python.py

```diff
@@ -528,13 +533,38 @@ def find_installation(self, interpreter, state, args, kwargs):
             if not python.found() and name_or_path in ['python2', 'python3']:
                 python = ExternalProgram('python', silent=True)
 
-            mlog.log('Program', python.name, 'found:',
-                     *[mlog.green('YES'), '({})'.format(' '.join(python.command))] if python.found() else [mlog.red('NO')])
+        if python.found() and want_modules:
+            for mod in want_modules:
+                p, out, err = mesonlib.Popen_safe(
+                    python.command + \
```

Comment:
__[Flake8]__ [E502] the backslash is redundant between brackets ([link](https://sider.review/gh/repos/19784232/pulls/5204#issue-4191315))

<sub>posted by Sider</sub>